### PR TITLE
New: Infrastructure Policies

### PR DESCRIPTION
**Pull request by ericrpowers**

### CODE REVIEW

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "security-groups",
    "component": "apotests",
    "pr-id": "2022",
    "commit-sha": "ca229592957563ad5f7d02ff31ba4e059fc7403f"
  },
  {
    "project": "security-groups",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "2ccd3d800929641baa918ab971ab9ae40191a434"
  },
  {
    "project": "security-groups",
    "component": "backend",
    "pr-id": "416",
    "commit-sha": "f0b310b0b94128e8313a78f931133753d8b81d33"
  },
  {
    "project": "security-groups",
    "component": "clad",
    "pr-id": "1137",
    "commit-sha": "1ea90ca6f1ef3a98d9b1d158113c31ff852daf5f"
  },
  {
    "project": "security-groups",
    "component": "gaiajs",
    "pr-id": "14",
    "commit-sha": "fda3086fd7d42a4aef9e56c0a557dc77fc1b8c48"
  },
  {
    "project": "security-groups",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "2f7cc3dbf95c0b2ee2449c9c6343b936d24e42f1"
  },
  {
    "project": "security-groups",
    "component": "underwater",
    "pr-id": "64",
    "commit-sha": "4d6a700d78856f4b033ddc7a086cfafb9b171493"
  },
  {
    "project": "security-groups",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "security-groups",
    "component": "backend",
    "pr-id": "416",
    "commit-sha": "7cc50fcddca0e29ee41e779fbacbcfad2cd012da"
  },
  {
    "project": "security-groups",
    "component": "clad",
    "pr-id": "1137",
    "commit-sha": "1ea90ca6f1ef3a98d9b1d158113c31ff852daf5f"
  },
  {
    "project": "security-groups",
    "component": "gaiajs",
    "pr-id": "14",
    "commit-sha": "fda3086fd7d42a4aef9e56c0a557dc77fc1b8c48"
  },
  {
    "project": "security-groups",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "b3263b019192f08d25d96ce377b12733129cf5ab"
  },
  {
    "project": "security-groups",
    "component": "underwater",
    "pr-id": "64",
    "commit-sha": "1ade7680915d81ee2db8cc8a8f0a3b7ed9d0f779"
  },
  {
    "project": "security-groups",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  },
  {
    "project": "security-groups",
    "component": "apotests",
    "pr-id": "2022",
    "commit-sha": "6855af0222a41b4df31ba971dd868360d19b178f"
  },
  {
    "project": "security-groups",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "2ccd3d800929641baa918ab971ab9ae40191a434"
  }
]
```

File: specs/infrastructurepolicy.spec [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    global_parameters:
+    - $filtering
+  extends:
+  - '@base'
+  - '@described'
+  - '@disabled'
+  - '@identifiable-not-stored'
+  - '@metadatable'
+  - '@named'
+  - '@propagated'
```
Comment (primalmotion) [inline specs/infrastructurepolicy.spec]: this doesn't propagate it seems

File: specs/infrastructurepolicy.spec [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  - '@fallback'
```
Comment (primalmotion) [inline specs/infrastructurepolicy.spec]: it's not be used as fallback

File: specs/infrastructurepolicy.spec [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    other ACL based mechanisms. They are not used in the identity-based network
+    authorization of Aporeto, but they can affect traffic flows in the underlying
+    infrastructure.
+  aliases:
+  - infrapol
+  - infrapols
+  get:
+    description: Retrieves the object with the given ID.
+    global_parameters:
+    - $propagatable
```
Comment (primalmotion) [inline specs/infrastructurepolicy.spec]: then you don't need that either :)

File: specs/root.spec [hunk 4]
```diff
@@ -359,6 +359,15 @@ relations:
   create:
     description: Creates a new import request.
 
+- rest_name: infrastructurepolicy
+  get:
+    description: Retrieves the list of infrastructure policies.
+    global_parameters:
+    - $filtering
+    - $propagatable
```
Comment (primalmotion) [inline specs/root.spec]: here too

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "security-groups",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "2ccd3d800929641baa918ab971ab9ae40191a434"
  },
  {
    "project": "security-groups",
    "component": "backend",
    "pr-id": "416",
    "commit-sha": "06c6a4921f753f78649dbb0fa34a53f85b7dbabd"
  },
  {
    "project": "security-groups",
    "component": "clad",
    "pr-id": "1137",
    "commit-sha": "1ea90ca6f1ef3a98d9b1d158113c31ff852daf5f"
  },
  {
    "project": "security-groups",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "a21db1342d6989fb581b56681c72811d8f542586"
  },
  {
    "project": "security-groups",
    "component": "underwater",
    "pr-id": "64",
    "commit-sha": "1ade7680915d81ee2db8cc8a8f0a3b7ed9d0f779"
  },
  {
    "project": "security-groups",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  },
  {
    "project": "security-groups",
    "component": "apotests",
    "pr-id": "2022",
    "commit-sha": "6855af0222a41b4df31ba971dd868360d19b178f"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "security-groups",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "2ccd3d800929641baa918ab971ab9ae40191a434"
  },
  {
    "project": "security-groups",
    "component": "backend",
    "pr-id": "416",
    "commit-sha": "32a027d1c714afaf97847a2f511f9fb98329a5c5"
  },
  {
    "project": "security-groups",
    "component": "clad",
    "pr-id": "1137",
    "commit-sha": "1ea90ca6f1ef3a98d9b1d158113c31ff852daf5f"
  },
  {
    "project": "security-groups",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "a21db1342d6989fb581b56681c72811d8f542586"
  },
  {
    "project": "security-groups",
    "component": "underwater",
    "pr-id": "64",
    "commit-sha": "a91d2c6ecc3e8eea96f57f432d2b69afb4b09815"
  },
  {
    "project": "security-groups",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  },
  {
    "project": "security-groups",
    "component": "apotests",
    "pr-id": "2022",
    "commit-sha": "6855af0222a41b4df31ba971dd868360d19b178f"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "security-groups",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "2ccd3d800929641baa918ab971ab9ae40191a434"
  },
  {
    "project": "security-groups",
    "component": "backend",
    "pr-id": "416",
    "commit-sha": "32a027d1c714afaf97847a2f511f9fb98329a5c5"
  },
  {
    "project": "security-groups",
    "component": "clad",
    "pr-id": "1137",
    "commit-sha": "1ea90ca6f1ef3a98d9b1d158113c31ff852daf5f"
  },
  {
    "project": "security-groups",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "a6a5310bde60a2b3ac0f9f7d8815bff1e977a550"
  },
  {
    "project": "security-groups",
    "component": "underwater",
    "pr-id": "64",
    "commit-sha": "94ffb33ae67e41769dd2910022b7c5c7055e2b5b"
  },
  {
    "project": "security-groups",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  },
  {
    "project": "security-groups",
    "component": "apotests",
    "pr-id": "2022",
    "commit-sha": "cf9b9ba48d6c023fa26cde68ab59d056d457ce4b"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "security-groups",
    "component": "apotests",
    "pr-id": "2022",
    "commit-sha": "e4c893e71f412f4dc65e1473c58535986bd3e827"
  },
  {
    "project": "security-groups",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "1c549f46ce8e2ef4fac45aa8d4fd4e62e0e6bc10"
  },
  {
    "project": "security-groups",
    "component": "backend",
    "pr-id": "416",
    "commit-sha": "a9acd7c0b8e58c0e3279abc297523908635959d9"
  },
  {
    "project": "security-groups",
    "component": "clad",
    "pr-id": "1137",
    "commit-sha": "1ea90ca6f1ef3a98d9b1d158113c31ff852daf5f"
  },
  {
    "project": "security-groups",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "9f60072982d587eaa7b01db8db654aa0451215cb"
  },
  {
    "project": "security-groups",
    "component": "underwater",
    "pr-id": "64",
    "commit-sha": "94ffb33ae67e41769dd2910022b7c5c7055e2b5b"
  },
  {
    "project": "security-groups",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "1c549f46ce8e2ef4fac45aa8d4fd4e62e0e6bc10"
  },
  {
    "project": "",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "9f60072982d587eaa7b01db8db654aa0451215cb"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "1c549f46ce8e2ef4fac45aa8d4fd4e62e0e6bc10"
  },
  {
    "project": "",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "72f29410a56cf840a4b0aa12a0f18f408590e161"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "416",
    "commit-sha": "38c761f9bc4f447c825495791b5e7ae0b1953f14"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "64",
    "commit-sha": "94ffb33ae67e41769dd2910022b7c5c7055e2b5b"
  },
  {
    "project": "",
    "component": "clad",
    "pr-id": "1137",
    "commit-sha": "1ea90ca6f1ef3a98d9b1d158113c31ff852daf5f"
  },
  {
    "project": "",
    "component": "apotests",
    "pr-id": "2022",
    "commit-sha": "77250f22931c33aba5af579dafb33477403fb75b"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "security-groups",
    "component": "underwater",
    "pr-id": "64",
    "commit-sha": "94ffb33ae67e41769dd2910022b7c5c7055e2b5b"
  },
  {
    "project": "security-groups",
    "component": "aws-lib",
    "pr-id": "11",
    "commit-sha": "bb9a9e18b4429ae7cf56dd0d51bd28f98a89705f"
  },
  {
    "project": "security-groups",
    "component": "apotests",
    "pr-id": "2022",
    "commit-sha": "77250f22931c33aba5af579dafb33477403fb75b"
  },
  {
    "project": "security-groups",
    "component": "wedge",
    "pr-id": "94",
    "commit-sha": "2446adb7077fc405a1eff7ad07b16769d23558a1"
  },
  {
    "project": "security-groups",
    "component": "backend",
    "pr-id": "416",
    "commit-sha": "38c761f9bc4f447c825495791b5e7ae0b1953f14"
  },
  {
    "project": "security-groups",
    "component": "clad",
    "pr-id": "1137",
    "commit-sha": "1ea90ca6f1ef3a98d9b1d158113c31ff852daf5f"
  },
  {
    "project": "security-groups",
    "component": "gaia",
    "pr-id": "338",
    "commit-sha": "72f29410a56cf840a4b0aa12a0f18f408590e161"
  }
]
```

**Pull request merged**